src/prototypes/draft-email-textarea/demo/demo.twig

```diff
@@ -0,0 +1,48 @@
+{# The `o-container` wrappers are here only for demo purposes #}
+<div class="o-container o-container--pad o-container--prose">
+	<div class="o-container__content o-rhythm o-rhythm--default o-rhythm--generous-headings">
+		{# Draft email message #}
+		<textarea
+			id="email-draft"
+			class="c-input u-pad-1 js-elastic-textarea"
+			spellcheck="false"
+			data-message-subject="{{ draftSubject }}"
+		>{{ draftMessage }}</textarea>
+
+		{# Copy and Draft buttons #}
+		{% embed '@cloudfour/objects/button-group/button-group.twig' with {
+			grow: true,
+			class: 'u-space-block-start-n2',
+			subject: draftSubject | url_encode,
+			message: draftMessage | url_encode
+		} only %}
+			{% block content %}
+				{% include '@cloudfour/components/button/button.twig' with {
+					label: 'Copy',
+					id: 'copy-btn',
+					content_start_icon: 'copy',
+				} only %}
+				{% include '@cloudfour/components/button/button.twig' with {
+					label: 'Draft email',
+					id: 'draft-email-btn',
+					class: 'c-button--secondary',
+					content_start_icon: 'envelope',
+					href: "mailto:info@cloudfour.com?subject=#{subject}&body=#{message}",
+				} only %}
+			{% endblock %}
+		{% endembed %}
+
+		{# "Copied" message shown when the "copy" btn is activated #}
+		<p
+			id="copied-action-message"
+			role="status"
```

Comment:
Based on my understanding, a `role` of `"status"` should be used for this message.

See: https://www.w3.org/WAI/WCAG22/Understanding/status-messages.html